.github/workflows/build-docs.yaml

```diff
@@ -10,6 +11,11 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v1
 
+      - name: Copy example-notebooks repository
+        run: |
+          git clone https://github.com/dandi/example-notebooks
+          mv ./example-notebooks ./docs/
```

Comment:
I actually prefer this. We use submodules in PyNWB and it's been a headache requiring special clone syntax. This is simpler and easier.

Reply:
Its been a while since I have used submodules so happy to look into it.  I filed an issue (https://github.com/dandi/handbook/issues/148) to perhaps expedite this pull request.

Reply:
it is quite easy really, let me know  if you need a hand with that.

It is just that without using them we would get "fluctuating" results of the doc build - it would depend on current version which has its pluses and minuses:

- `+`: we would always be "up to date" with notebooks repo since we would always use current master
-  `-`: as a result we might get sporadic docs build failures if something in current state of notebooks would trigger failing their rendering/inclusion
    - and it would require then adjusting build script to point to specific commit etc making it "hidden" 

but in either of the cases, in the long run we better also trigger rebuilding of the handbook whenever notebooks collection change.

- without `git submodule` -- it would be matter of just triggering rebuild workflow
- with `git submodule` -- it would require PR or at least a commit to update the state of the submodule.

So overall, let's may be proceed without submodules and see where we end up ;-)